plugin/eleline.vim

```diff
@@ -230,9 +249,15 @@ function! s:StatusLine() abort
   let l:tags = '%{exists("b:gutentags_files") ? gutentags#statusline() : ""} '
   let l:lcn = '%{ElelineLCN()}'
   let l:coc = '%{ElelineCoc()}'
-  let l:vista = '%#ElelineVista#%{ElelineVista()}%*'
+  if empty(get(b:, 'vista_nearest_method_or_function', ''))
+    let l:lsp = '%{ElelineNvimLsp()}'
```

Comment:
Since it's a neovim-only feature, we should check if we are using neovim.

Reply:
added has('nvim-0.5')